src/mkdocstrings_handlers/python/templates/material/_base/docstring/parameters.html

```diff
@@ -60,6 +60,14 @@
         <div class="doc-md-description">
           {{ parameter.description|convert_markdown(heading_level, html_id) }}
         </div>
+        {% if parameter.default %}
+          <p>
+            <b>{{ lang.t("DEFAULT:") }}</b>
```

Comment:
I'd prefer a normal case Default:

```suggestion
            <b>{{ lang.t("Default:") }}</b>
```

If "Default:" does not exist in the language macros it should be added please :slightly_smiling_face: 
Can you also make the same change in the readthedocs templates?

Reply:
Done.